python/feast_spark/client.py

```diff
@@ -51,6 +47,45 @@
 )
 
 
+def stage_entities_to_bq_with_partition(
```

Comment:
can we avoid duplication by adding `partition` as an optional argument in existing  feast.staging.entities file ?

Reply:
The issue is that, we would have to make this change in the feast repository, and we are having issue publishing Python SDK for branch 0.9 at the moment.